<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>781</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>显示接收的图片文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_gender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_gender_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_national">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>national:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_national_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_birthday">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>birthday:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_birthday_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_address">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_address_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_idnumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>idnumber:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_maker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>maker:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_start_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>88</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>start_date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_end_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>end_date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_idnumber_show">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_maker_show">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_startdate_show">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>320</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_enddate_show">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>服务器IP：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>192.168.2.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>连接(&amp;L)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>缩放</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
